.github/workflows/release.yml

```diff
@@ -0,0 +1,282 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+
+jobs:
+  build-and-release:
+    name: Build ${{ matrix.target }}
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # Linux x86_64
```

Comment:
@Rqnsom take note that Linux ARM is missing. 
I couldn't fix an openssl error in the CI.

Should I spend extra time on it or you recon it's ok to not support it?

Reply:
Added support. Resolving